test/quantization/test_quantize.py

```diff
@@ -1247,6 +1248,36 @@ def forward(self, x):
     def test_leaky_relu(self):
         self._test_activation_op_impl(nn.LeakyReLU, nnq.LeakyReLU, {'negative_slope': 0.1, 'inplace': False})
 
+
+class TestEagerModeQATOps(QuantizationTestCase):
+    def test_fixed_qparam_ops(self):
+        class M(torch.nn.Module):
+            def __init__(self):
+                super().__init__()
+                self.sigmoid = torch.nn.Sigmoid()
+                self.hardsigmoid = torch.nn.Hardsigmoid()
+                self.tanh = torch.nn.Tanh()
+                self.quant = QuantStub()
+                self.dequant = DeQuantStub()
+
+            def forward(self, x):
+                x = self.quant(x)
```

Comment:
Does the approach here also work for cases where the scale and zero-point are not fixed? i.e for a hardTanh, the scale and zero-point depend on the arguments to init.

Reply:
maybe, this allows to provide special fake quantize(activation_post_process) for a specific pattern (module/functional/torch op), could you write down the details, is it like following?
```
fake_quant = HardTanhFakeQuantize(hardtanh_instance.params)
```
I think this should be too hard to support in current implementation.

Reply:
Actually for hardtanh, our implementation calls qclamp: https://github.com/pytorch/pytorch/blob/87a4baf616565aecd21884754772a0f03babfe2d/aten/src/ATen/native/quantized/cpu/qclamp.cpp#L24, which doesn't need LUTs. In this case, we should treat it like ReLU, i.e there is no fake-quant or observer needed at the output as there is no quantization that occurs inside the op.

Reply:
yeah this is how it is implemented right now: https://github.com/pytorch/pytorch/blob/master/torch/quantization/fx/quantization_patterns.py#L512